wiki/User_title/zh.md

```diff
@@ -0,0 +1,35 @@
+---
+tags:
+  - 论坛头衔
+  - 等级
+  - 头衔
+  - 用户头衔颜色 # 如果什么时候有了该页面则可以删除这个tag
+  - 用户组颜色
+
+---
+
+# 用户头衔
+
+![](img/dev.png "在Tom94的主页上方可以看到\"osu!dev\" 头衔")
+
+**用户头衔**通常是一个位于用户主页的小标签，该标签由 [osu!team](/wiki/People/The_Team) 设置。大多数头衔都代表了一个[用户组](/wiki/People/The_Team#user-groups)的成员身份。
+除此之外，还有一些[特殊头衔](/wiki/People/Users_with_unique_titles)，能获得这些头衔的用户都有着杰出的贡献或成就。
+
+通常这些头衔的颜色会与用户组颜色一致，颜色由用户组的成员设置。如果没有设置则默认为白色。
+
+## 历史
+
+### 发帖数头衔
+
+![位于Damnae已发帖数上方的\"Star Shooter\" 头衔](img/star-shooter.png " 位于Damnae已发帖数上方的\"Star Shooter\" 头衔")
+
+<!-- 这个老旧的历史能在网站时光机中看见： https://web.archive.org/web/20131107133826/https://osu.ppy.sh/forum/t/141240 -->
+
```

Comment:
```suggestion
```

Reply:
This note exists in the english ver too.